test/spec/modules/lemmaBidAdapter_spec.js

```diff
@@ -309,6 +313,77 @@ describe('lemmaBidAdapter', function() {
         expect(data.imp[0]['video']['w']).to.equal(videoBidRequests[0].mediaTypes.video.playerSize[0]);
         expect(data.imp[0]['video']['h']).to.equal(videoBidRequests[0].mediaTypes.video.playerSize[1]);
       });
+      describe('setting imp.floor using floorModule', function() {
+        /*
+        Use the minimum value among floor from floorModule per mediaType
+        If params.bidFloor is set then take max(floor, min(floors from floorModule))
+        set imp.bidfloor only if it is more than 0
+        */
+
+        let newRequest;
+        let floorModuleTestData;
+        let getFloor = function(req) {
+          return floorModuleTestData[req.mediaType];
+        };
+
+        beforeEach(() => {
+          floorModuleTestData = {
+            'banner': {
+              'currency': 'USD',
+              'floor': 1.50
+            },
+            'video': {
+              'currency': 'USD',
```

Comment:
For currency please use `AUD`, in all your tests you have `params.currency: 'AUD'`, so following if condition is never true:

```
if (typeof floorInfo === 'object' && floorInfo.currency === impObj.bidfloorcur && !isNaN(parseInt(floorInfo.floor)))
```

Because `fllorInfo.currency` in your tests is always `USD` while `impOb.jbidfloorcur` is always `AUD`

Reply:
Made suggested changes.